server/pkg/thread/comment_test.go

```diff
@@ -10,14 +11,30 @@ func TestComment_CommentType(t *testing.T) {
 	cid := NewCommentID()
 	uid := NewUserID()
 	c := "xxx"
+	tim, _ := time.Parse(time.RFC3339, "2021-03-16T04:19:57.592Z")
 
 	got := Comment{
-		id:      cid,
-		author:  uid,
-		content: c,
+		id:        cid,
+		author:    uid,
+		content:   c,
+		createdAt: tim,
 	}
 
 	assert.Equal(t, cid, got.ID())
 	assert.Equal(t, uid, got.Author())
 	assert.Equal(t, c, got.Content())
+	assert.Equal(t, tim, got.CreatedAt())
+}
+
+func TestComment_SetContent(t *testing.T) {
+	c := "xxx"
+	got := Comment{}
+
+	got.SetContent(c)
+	assert.Equal(t, got.Content(), c)
+}
+
+func TestComment_CreatedAt(t *testing.T) {
+	var got *Comment = nil
+	assert.Equal(t, time.Time{}, got.CreatedAt())
```

Comment:
```suggestion
	var got *Comment = nil
	assert.Equal(t, time.Time{}, got.CreatedAt())
	got = &Comment{ID: NewID()}
	assert.Equal(t, got.id.Timestamp(), got.CreatedAt())
```

Please add more test cases as they are missing.

Reply:
Done!